packages/connect-form/src/components/advanced-options-tabs/advanced-tab/url-options-table.tsx

```diff
@@ -164,7 +152,7 @@ function UrlOptionsTable({
               datum.name ? `${datum.name}-table-row` : 'new-option-table-row'
             }
           >
-            <Cell>
+            <Cell className={optionNameCellStyles}>
```

Comment:
I asked this last time, but didn't get an answer: why are we using this table here at all? Seems like we are complicating layout both visually and from the code perspective, and the only thing this adds is the table headers that just duplicate the placeholders in the input fields

Reply:
Yeah, this makes sense probably ok to keep it for now